frontend/src/robokop_config.json

```
@@ -0,0 +1,4 @@
+{
+    "brand": "robokop",
+    "title": "Robokop"
```

Just wanted to check that you meant to capitalize one (the title) and not the other!

---

It's a little weird. I wanted the title to be capitalized because I'm putting that text directly into the page. The brand is more of a variable, so I didn't want it capitalized there. These config files are going to have a lot more in them once we start adding more styling differences between the two brands, but this is just the base logic.

---

ROBOKOP is an acronym.
```suggestion
    "title": "ROBOKOP"
```